### ISSUE TITLE
Autofold mess with search

### ISSUE THREAD

**Issue by Turgon-Aran-Gondolin** [bobmagicii/vscode-auto-fold#8]
When using search, if the item is in a folded area, sometimes vs code won't unfold it, instead it just highlight the dots in the end of the line. Not sure if that's because of this plugin but I didn't experience it in the original vscode interface.

**Comment by bobmagicii** [comment]
i do not think so - these past few days, and it was actually really bad today, just normally folding and unfolding things has been really hard. i think they broke something recently.
